src/napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -235,6 +261,10 @@ def _update_range(self):
             self.totslice_label.setAlignment(Qt.AlignmentFlag.AlignLeft)
             self._update_slice_labels()
 
+            self.thickness_slider.setMinimum(0)
+            self.thickness_slider.setMaximum(self.dims.nsteps[self.axis] * 2)
```

Comment:
Where does the *2 come from? If I understand, we're taking a number of slices for the e.g. mean projection, instead of just one. So shouldn't the max be the total number of slices?
I tested this and it doesn't appear to be the case, when I set the slider to 59 for cells3d i still see differences as I scrub the stack, so obviously i'm not getting it.

Reply:
If you set the thickness to the length of the dim and your slider is all the way to the left, your thick slice reaches to the middle of the dim, since the other half of the thick slice extends into the nothingness on the left. by setting the max to *2, if you have max thickness you are guaranteed to cover the full dimension no matter what.

Reply:
Oh dang. does that mean i'm getting 0's averaged in when the slider is all the way right or left?
I wonder if we can abstract this better so it's less confusing...

Reply:
Nope, those are `nan`s and they are all ignored :)